assets/css/src/components-form-editor/_sidebar.scss

```diff
@@ -105,6 +105,11 @@
   border: 1px solid #eee;
 }
 
+.form-placement-option-list {
+  display: flex;
+  flex-direction: row;
+}
+
 .form-placement-option {
```

Comment:
This shouldn't have any left/right margin but rather add `justify-content: space-between` and `flex-wrap: wrap;` to `.form-placement-option-list`